changelog.d/20228_fix-log-to-metric-set-supported.fix.md

```diff
@@ -0,0 +1,3 @@
+Fixed an issue where the log_to_metric transform with all_metrics=true config failed to convert properly-formatted 'set'-type events into metrics.
+
+author: pabloem
```

Comment:
```suggestion
authors: pabloem
```

Just realized this should be `authors:` which should also tell the spell checker to ignore the line.